docs/modules/control/move_to_a_pose_control/move_to_a_pose_control.rst

```diff
@@ -13,34 +13,39 @@ Constructor
 ~~~~~~~~~~~
 
 .. code-block:: ipython3
+
```

Comment:
I think the most variable doc for users is about the control logic and methematics of the PathFinderController which are implemented here:
https://github.com/AtsushiSakai/PythonRobotics/blob/7034d5ff9eb19eff561f84c806ec591b2d0c8e26/Control/move_to_pose/move_to_pose.py#L73-L78
Is it possible to add the doc here?

Reply:
The document for the logic added.